asclepias_broker/api/ingestion.py

```diff
@@ -452,3 +452,11 @@ def update_metadata(relationship: Relationship, payload):
         rel_metadata.update(
             {k: v for k, v in payload.items()
              if k in ('LinkPublicationDate', 'LinkProvider')})
+
+
+def update_group_metadata(identifier, payload):
```

Comment:
It's better to use simple/primitive values as parameters to functions, and in this case `identifiers` is a dictionary with a specific format, which means that you would have to document it. 

Maybe it's better to have something like `def update_group_metadata(id_value, id_scheme, payload):`